modules/api/src/main/java/org/apache/ignite/configuration/schemas/network/NodeFinderConfigurationSchema.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.configuration.schemas.network;
+
+import org.apache.ignite.configuration.annotation.Config;
+import org.apache.ignite.configuration.annotation.Value;
+
+/** Node finder configuration. */
+@Config
+public class NodeFinderConfigurationSchema {
+    /** Node finder type. */
+    @Value(hasDefault = true)
+    public final String type = NodeFinderType.STATIC.name();
+
+    /** Cluster nodes. */
+    @Value(hasDefault = true)
+    public final String[] netClusterNodes = new String[0];
+
+    /** Port start. */
+    @Value(hasDefault = true)
```

Comment:
Is this a good default value for port? I don't think so

Reply:
Maybe there should not be a default value at all?